src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,190 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    private static class Node<T> {
+        private T item;
+        private Node<T> next;
+        private Node<T> prev;
+
+        public Node(Node<T> prev, T item, Node<T> next) {
+            this.prev = prev;
+            this.item = item;
+            this.next = next;
+        }
+    }
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (head == null) {
```

Comment:
this method may be simplified, try smt like this: 

 - // create newNode object  like (head, value, null);
 - // if  your list is empty then tail = newNode, else head.next = newNode
 - // in any case head = newNode and size increase